rasa/core/validator.py

```diff
@@ -57,13 +58,43 @@ def verify_intents(self, ignore_warnings: bool = True) -> bool:
 
         return everything_is_alright
 
+    def verify_example_repetition_in_intents(
```

Comment:
this method is about Rasa NLU. I think we should move `validator.py` to rasa directory then. @tmbo what do you think?